e2e-spock-geb compilation error in GebConfig.groovy

**Describe the bug**
The quickstarter-generated pipeline fails with a compilation error in GebConfig.groovy.

**To Reproduce**
Steps to reproduce the behavior:
1. Provision an EDP project
2. Provision a e2e-spock-geb component
3. The ods-qs-demo-app-end-to-end-3x job completes correctly.
4. The demo-app-end-to-end-master job shows a compilation error in GebConfig.groovy and finishes as unstable.

**Expected behavior**
GebConfig.groovy compiles correctly and the job finishes succesfully

**Screenshots**
![image](https://user-images.githubusercontent.com/13270880/99660737-0f320f80-2a63-11eb-877e-4b9f343abaa2.png)

**Affected version (please complete the following information):**
 - OpenShift: OKD version:
OpenShift Master: unknown
Kubernetes Master: v1.11.0+d4cacc0
OpenShift Web Console: v3.11.0+ea42280

 - OpenDevStack 3.x

**Log Output (ensure to remove any confidential information like tokens, project names, etc.**
```
> Task :installation-groovy
Picked up JAVA_TOOL_OPTIONS: -XX:+UnlockExperimentalVMOptions -Dsun.zip.disableMemoryMapping=true
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.codehaus.groovy.vmplugin.v7.Java7$1 (file:/home/jenkins/.gradle/caches/modules-2/files-2.1/org.codehaus.groovy/groovy/2.5.6/6936e700f0fb1b50bac0698ada4347a769d40199/groovy-2.5.6.jar) to constructor java.lang.invoke.MethodHandles$Lookup(java.lang.Class,int)
WARNING: Please consider reporting this to the maintainers of org.codehaus.groovy.vmplugin.v7.Java7$1
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release

[0K[1mDemoInstallation[m

[0K[1m  Test [22mclassMethod[31m FAILED[31m

  geb.error.UnableToLoadException: Unable to load configuration @ 'file:/tmp/workspace/t2-cd/t2-cd-demo-app-end-to-end-master/build/resources/installation/GebConfig.groovy' (with environment: )

  Caused by: org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
  script1605776752050542709543.groovy: 11: unable to resolve class WebClient 
   @ line 11, column 7.
           @Override
           ^
  
  script1605776752050542709543.groovy: 13: unable to resolve class WebClient 
   @ line 13, column 21.
               WebClient webClient = super.newWebClient(version);
                         ^
  
  2 errors

[m

[0K[1;31mFAILURE: [39mExecuted 1 tests in 5s (1 failed)[m


1 test completed, 1 failed
There were failing tests. See the report at: file:///tmp/workspace/t2-cd/t2-cd-demo-app-end-to-end-master/build/reports/tests/installation-groovy/index.html
```

**Additional context**
It cannot find some classes. Apparently it lacks the needed imports.

--- a/e2e-spock-geb/files/src/test/resources/GebConfig.groovy
+++ b/e2e-spock-geb/files/src/test/resources/GebConfig.groovy
@@ -1,4 +1,5 @@
 import com.gargoylesoftware.htmlunit.BrowserVersion
+import com.gargoylesoftware.htmlunit.WebClient
 import org.openqa.selenium.htmlunit.HtmlUnitDriver
 import org.openqa.selenium.Proxy
 

